src/flb_utils.c

```diff
@@ -744,23 +746,38 @@ int flb_utils_write_str(char *buf, int *off, size_t size,
 
             is_valid = FLB_TRUE;
             for (b = 0; b < hex_bytes; b++) {
```

Comment:
nitpick : would it be possible to use more expressive names such as utf_sequence_number or utf_sequence_length?

Reply:
Sounds good. I like utf_sequence_length, utf_sequence_number. Maybe `i` could also be changed to buf_byte_index

Reply:
done